optaplanner-core/src/test/java/org/optaplanner/core/impl/solver/DefaultSolverTest.java

```diff
@@ -127,13 +127,15 @@ public void zeroEntity() {
         Solver<TestdataChainedSolution> solver = solverFactory.buildSolver();
 
         TestdataChainedSolution solution = new TestdataChainedSolution("1");
-        solution.setChainedEntityList(Collections.singletonList(new TestdataChainedEntity("3")));
+        solution.setChainedEntityList(Collections.EMPTY_LIST);
         solution.setChainedAnchorList(Collections.singletonList(new TestdataChainedAnchor("4")));
 
         try {
             solver.solve(solution);
         } catch (Exception exception) {
-            assertTrue(exception instanceof IllegalStateException);
+            assertEquals(true,exception instanceof IllegalStateException);
```

Comment:
If you assert logical expressions, use just `assertTrue` or `assertFalse`. Let's keep the `assertEquals` for something more complex.

Reply:
I was trying to stick to the way assertions were done in the test before: e.g.
`assertEquals(false, solution.getScore().isSolutionInitialized());` in `solve()`, `solveStopsWhenUninitialized()` or `solveStopsWhenPartiallyInitialized()` tests.

Reply:
@Fonifan please take it just as recommendation, from my point of view, it doesn't block merging.

As we talk about consistency, could you please add a space after a comma in `true,exception` ?

Reply:
Thanks for noticing, fixed. :+1: